src/libraries/System.Text.Json/tests/Serialization/ExtensionDataTests.cs

```diff
@@ -250,6 +250,44 @@ public static void IgnoredDataShouldNotBeExtensionData()
             Assert.Null(obj.MyOverflow);
         }
 
+
+        private class ClassWithExtensionData
+        {
+            [JsonExtensionData]
+            public Dictionary<string, object> Overflow { get; set; }
+        }
+
+        public class DictionaryOverflowConverter : JsonConverter<Dictionary<string, object>>
+        {
+            public override Dictionary<string, object> Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+            {
+                throw new NotImplementedException();
+            }
+
+            public override void Write(Utf8JsonWriter writer, Dictionary<string, object> value, JsonSerializerOptions options)
+            {
+                writer.WriteString(JsonEncodedText.Encode("MyCustomOverflowWrite"), "OverflowValueWrite");
```

Comment:
```suggestion
                writer.WriteString("MyCustomOverflowWrite", "OverflowValueWrite");
```

We can keep this simple for testing purposes. In real usage this is only helpful if the encoded text is statically cached.